private/buf/cmd/buf/command/beta/convert/testdata/bin_json/payload.bin

```diff
@@ -0,0 +1 @@
+7
```

Comment:
checking in a binary?

Reply:
[This is the testing strategy of this repo and already exists in this repo](https://github.com/bufbuild/buf/blob/de3366af16161cea0f7f502d9df81600eb7bd920/private/buf/cmd/buf/testdata/convert/descriptor.plain.bin)

I don't like it either so: https://github.com/bufbuild/buf/issues/1469

Reply:
too easy, resolved